src/bump-stdlibs.jl

```diff
@@ -119,16 +122,20 @@ function _bump_single_stdlib(stdlib::StdlibInfo, config::Config, state::State)
                     cd(joinpath(temp_dir, "FORK")) do
                         run(`git checkout $(config.julia_repo_target_branch)`)
                         assert_current_branch_is(config.julia_repo_target_branch)
-                        pr_title_without_emoji = "Bump the $(stdlib.name) stdlib from $(stdlib_current_commit_in_upstream_short) to $(stdlib_latest_commit_short)"
+                        pr_title_without_emoji = "Bump the $(stdlib.name) stdlib from $(stdlib_current_commit_in_upstream_short) to $(stdlib_latest_commit_short) on the `$(config.julia_repo_target_branch)` branch"
                         pr_title = "🤖 $(pr_title_without_emoji)"
```

Comment:
I think a PR title like this would be more standard and concise
```
[release-1.8] 🤖 Bump the Pkg stdlib from ufd8g7df7 to 8fdjfd98g
```

Reply:
Should we still put the robot emoji at the beginning of the title?

Reply:
I went ahead and kept the robot emoji at the very beginning; I figure this makes it easy for people to very quickly visually identify a PR as being from a bot.

Reply:
Sounds good